app/views/contexts/feedback_form_context.py

```diff
@@ -14,7 +14,13 @@ def build_feedback_context(question_schema, form):
         "sign_out_url": url_for("session.get_sign_out"),
     }
 
-    answer_ids = (answer["id"] for answer in question_schema["answers"])
+    answer_ids = []
+    for answer in question_schema["answers"]:
+        answer_ids.append(answer["id"])
+        if answer["type"] in ("Checkbox", "Radio"):
+            for option in answer["options"]:
+                if "detail_answer" in option:
+                    answer_ids.append(option["detail_answer"]["id"])
```

Comment:
I can see why this has been done, but the duplication in this file and the question context is not ideal. We could replace most of this file with a call to `build_question_context`, and append what we need before returning the context:

```python
def build_feedback_context(question_schema, form):
    block = {
        "question": question_schema
    }
    context = build_question_context(block, form)
    context["hide_sign_out_button"] = False
    context["question"] = question_schema
    context["sign_out_url"] = url_for("session.get_sign_out")

    return context
```

Reply:
Calling `build_question_context` now, thanks!